browser/resources/settings/brave_ipfs_page/brave_ipfs_page.js

```diff
@@ -18,11 +18,22 @@ Polymer({
     WebUIListenerBehavior,
     PrefsBehavior
   ],
-
+  
+  /** 
+   * Keep it same as in IPFSResolveMethodTypes
+   * in brave\components\ipfs\ipfs_constants.h */
+  IPFSResolveMethodTypes: {
```

Comment:
Maybe add another comment to brave\components\ipfs\ipfs_constants.h referencing this as well?

Reply:
done